wbooks-api/app/models/rent.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+class Rent < ApplicationRecord
+  belongs_to :user
+  belongs_to :book
+
+  validates :from, :to, presence: true
```

Comment:
While this is fine, you might have some problems naming a column 'from' since it's a reserved word in SQL. It might work but I've had problems because of that. I'd add a prefix to both from and to instead